_summaries/bjorck2018understanding_1.md

```diff
@@ -1,19 +1,63 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{bjorck2018understanding}}
 # Please fill out info below
-author: # Your GitHub id
+author: # timchen0618
 score: # How did you like this paper 0(dislike) to 10(love)
 ---
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+## Background
+Normalizing the input data to zero-mean and constant standard deviation has long been recognized as good practice in training neural networks. 
+Batch normalization (BN) extend this idea to the input of each layer and achieve immense success in various areas. 
+It enables faster training, larger learning rates, and higher testing accuracy (and better generalizability). 
+However, the reason for the improvement is to be debated. 
+(e.g. the paper that propose BN said it was solving "internal covariate shift", but some suggested otherwise)
+
+## Core idea 
+The paper pointed out that BN is beneficial to training neural networks mainly because it allow for larger learning rates. 
+It also showed empirically that larger learning rates in networks without BN could lead to strong gradient updates, exploding activations and diverging loss . The same cannot be observed for neural networks with batch normalization, since the activation will be contained within a certain range, hence the stable loss and gradient. 
```

Comment:
I am a little confused here, in my understanding, I think large learning makes networks without BN diverge, but since networks with BN still converges with large learning rate, and by the fact that larger learning rate cause more noise when using SGD method, which makes prevents network from getting trapped in sharp minima, and get better generalization. Sorry if I understand this wrong. I guess maybe it will be nice if you can write more details here.

Reply:
I think what I am talking about here is why networks with BN still converges on large learning rates. They first showed empirically that BN is beneficial because of the large learning rate they use, and later showed why such large rates can be used. The generalizability is more of the theoretical reason why BN is beneficial, but yeah maybe I should include that as well. (Was gonna put it in later sections)